index.html

```diff
@@ -1,21 +1,66 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>Page Title</title>
-    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <div class="container">
-      <header></header>
-      <section class="section-1">
-        <h1>I'm Marine Glad to see you</h1>
-        <p>
-          I'm a software Developer! I can help you build aproduct, feature or
-          website look through some of my work and exprience! if you like what
-          you se & have aproject you need, don't hestiate to contact me
-        </p>
-      </section>
+      <main>
+        <header>
+          <div class="logo">
+            <a href="#">
+              <img
+                src="Asset/My Logo.png"
+                alt="log for My portifolio"
+                class="logo"
+            /></a>
+          </div>
+          <nav>
+            <img src="Asset/hamburger.png" alt="Menu Icon" class="menu-icon" />
+          </nav>
+        </header>
+
+        <section class="Content">
+          <h1>I'm Melkamu Alemawu Ayen!</h1>
+          <p>
+            I'm a software developer! I can help you build a product, feature or
+            website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hestiate to
+            contact me.
+          </p>
+          <div class="socialMediaGroup">
+            <h4 class="connect">LET'S CONNECT</h4>
+            <ul class="social">
+              <li>
+                <a href=""><img src="Asset/twitter.png" alt="" /></a>
+              </li>
+              <li>
+                <a href=""><img src="Asset/Linkedin icon.png" alt="" /></a>
+              </li>
+              <li>
+                <a href=""><img src="Asset/medium.png" alt="" /></a>
+              </li>
+              <li>
+                <a href=""><img src="Asset/vector3.png" alt="" /></a>
+              </li>
+              <li>
+                <a href=""><img src="Asset/angellist.png" alt="" /></a>
+              </li>
+            </ul>
```

Comment:
Good job on using the correct semantic html but Please make sure to align this items correctly as requested before , I suggest that you remove the default padding-left from ul tag and justify-content start rather than center this will fix this issue